### ISSUE TITLE
NTR: Pipecolic acid high in blood (Lab: chemistry: amino compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8531]
New term request Pipecolic acid high in blood (f306): Hyperpipecolatemia  UMLS: 4013944

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000268
- label: Elevated circulating pipecolic acid concentration
- parents: [Abnormal circulating monocarboxylic acid concentration (HP:0010996)](https://hpo.jax.org/app/browse/term/HP:0010996)
- pmid: 
- def: The concentration of pipecolic acid in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8531
- synonyms: Pipecolic acid high in blood(NONE)|Hyperpipecolatemia(NONE)
